bin/ckan_pycsw.py

```diff
@@ -14,6 +14,12 @@
 import pycsw.core.config
 import pycsw.core.admin
 
+from prometheus_client import CollectorRegistry, Gauge, push_to_gateway
+
+
+registry = CollectorRegistry()
```

Comment:
Seems like this is [necessary](https://github.com/prometheus/client_python/blob/09a5ae3/prometheus_client/exposition.py#L486), fair enough :)